packages/phoenix-evals/src/phoenix/evals/models/openai.py

```diff
@@ -278,9 +278,29 @@ def _init_rate_limiter(self) -> None:
         )
 
     def _build_messages(
-        self, prompt: str, system_instruction: Optional[str] = None
+        self, prompt: str | Audio, audio_format: AudioFormat = None, system_instruction: Optional[str] = None
```

Comment:
this is not off of main it seems

Reply:
also we can't use the `type1 | type2` syntax in Phoenix, as our lowest supported Python version does not support it